pkg/storage/minio/minio.go

```diff
@@ -276,3 +277,16 @@ func (c *Client) PlaceFiles(buckets []string, prefix string) error {
 	}
 	return nil
 }
+
+// GetValidBucketName returns a minio-compatible bucket name
+func (c *Client) GetValidBucketName(parentType string, parentName string) string {
+	bucketName := fmt.Sprintf("%s-%s", parentType, parentName)
+	if len(bucketName) <= 63 {
+		return bucketName
+	}
+
+	h := fnv.New32a()
+	h.Write([]byte(bucketName))
+
+	return fmt.Sprintf("%s-%d", bucketName[:52], h.Sum32())
```

Comment:
Are we sure that string will always be with length above 52? As if not it'll probably panic

Reply:
At this point bucketName is above 63, there is a check in line 284